packages/providers/src/operations.graphql

```diff
@@ -4,6 +4,31 @@
 # generate `operations.ts` from this file.
 
 # Fragments
+
+fragment transactionStatusFragment on TransactionStatus {
+  type: __typename
+  ... on SubmittedStatus {
+    time
+  }
+  ... on SuccessStatus {
```

Comment:
If it's of any use, you can now get all the receipts on either the Success or Failure status

Reply:
Thanks for the info! I created an issue for that: #1530.